<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="P3A-HQ3PXJ2" ignored="0" name="Regression" tests="1" failures="1" timestamp="23 Sep 2019 04:19:00 GMT" time="0.082" errors="0">
  <testcase name="f" time="0.034" classname="testngFeatures.DemoSoftAssert">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [12] but found [10],
	Error expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [12] but found [10],
Error expected [true] but found [false]
at testngFeatures.DemoSoftAssert.f(DemoSoftAssert.java:17)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- Regression -->
